pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -164,6 +165,58 @@ var _ = Describe("SRIOV HostDevice", func() {
 			Expect(devices, err).To(Equal([]api.HostDevice{expectHostDevice1}))
 		})
 
+		DescribeTable("create two devices with custom guest PCI address",
+			func(iface1GuestPCIAddress, iface2GuestPCIAddress string) {
+				iface1 := newSRIOVInterface(netname1)
+				iface1.PciAddress = iface1GuestPCIAddress
+
+				iface2 := newSRIOVInterface(netname2)
+				iface2.PciAddress = iface2GuestPCIAddress
+
+				var guestPCIAddress1 *api.Address
+				var guestPCIAddress2 *api.Address
+
+				var err error
+				if iface1.PciAddress != "" {
+					guestPCIAddress1, err = device.NewPciAddressField(iface1.PciAddress)
+					Expect(err).NotTo(HaveOccurred())
+				}
+
+				if iface2.PciAddress != "" {
+					guestPCIAddress2, err = device.NewPciAddressField(iface2.PciAddress)
+					Expect(err).NotTo(HaveOccurred())
+				}
```

Comment:
I would pass `api.Address` instead of PCI address strings, instead of these checks.
You define some fixed `api.Address` objects and pass them to each of the DescribeTabe Entry:
```
var (
    pciAddress1 = api.Address{Domain: "0x0000", Bus: "0x20", Slot: "0x00", Function: "0x0"}
    pciAddress2 = api.Address{Domain: "0x0000", Bus: "0x20", Slot: "0x01", Function: "0x0"}
)
```

Reply:
Now the DescribeTable func receives:
```go
func(iface1, iface2 v1.Interface)
```

Same as the table testing bootOrder.